backend/app/graphql/all_users.py

```diff
@@ -0,0 +1,39 @@
+from ..services.implementations.user_service import UserService
+import graphene
+import logging
+from .types import QueryList
+
+
+class UserType(graphene.ObjectType):
+    name = graphene.String()
+    email = graphene.String()
+    role = graphene.String()
+
+
+class AllUsersQuery(QueryList):
```

Comment:
Normally, `QueryList`s would be a list of queries rather than a single query. Might be better to create a separate `UserQueries` class which is a `QueryList`, and add `class AllUsersQuery(Query):` as a `all_users` field on the `UserQueries` class